packages/dev/test/HttpWrapper.test.ts

```diff
@@ -29,16 +29,10 @@ describe('HttpWrapper', () => {
     await expect(res).to.be.eventually.rejectedWith({ error: 'connect ECONNREFUSED 127.0.0.1:44321' })
   })
 
-  it('should timeout after specified time', async () => {
-    // this test abuses the fact that a local ganache is slow, and should take over 1ms to respond even if it's local
-    const http = new HttpWrapper({ timeout: 1 })
-    const res =
-      // @ts-ignore
-      http.sendPromise(web3.currentProvider.host, {
-        jsonrpc: '2.0',
-        method: 'net_version',
-        id: 123
-      })
-    return await expect(res).to.be.eventually.rejectedWith('timeout of 1ms exceeded')
+  it('should pass timeout to provider', async () => {
+    // This test should be removed. It checks axios functionality.
```

Comment:
lets remove it then; asserting a value is not cool
also, why can't you keep the expect.to.be.reverted check?

Reply:
https://github.com/opengsn/gsn/blob/995647ebf8e34ac183d5b99c06c385bc1995d6dd/packages/dev/test/HttpWrapper.test.ts#L33

This test used to rely on a race condition that worked with ganahce since it's slow. Now it doesn't, since hardhat is quicker to respond.

Reply:
I still think we should remove this test. it is useless